css/index.css

```diff
@@ -243,44 +243,53 @@ footer {
 
     header {
         flex-wrap: wrap;
-        height: 100%;
-        width: 100%;
+        justify-content: center;
     }
 
     header nav {
-        width: 93%;
+        width: 100%;
         flex-direction: column;
+        display: flex;
+        margin: 0;
         align-items: center;
-        justify-content: center;
-        text-align: center;
-        font-size: 24px;
-        margin-top: 5px;
     }
 
     header nav a {
-        border: 1px solid black;
-        width: 398px;
-        height: 50px;
-        align-items: center;
+         margin: 0;
```

Comment:
I hope the solution code/lecture has been helpful in gaining you insight into this material. Overall, your progression and code are looking great.